source/content/plugins-known-issues.md

```diff
@@ -745,26 +745,31 @@ ___
 
 ## [WP All Import / Export](http://www.wpallimport.com/)
 
-<ReviewDate date="2018-10-30" />
+<ReviewDate date="2020-06-15" />
 
 **Issue 1:** Large batch processes can fail if they take longer than the platform will allow. See [Timeouts on Pantheon](/timeouts) for more information.
 
 **Solution:** To avoid hitting a timeout, you can try:
+- Clean up temporary files
+- Lower the chunk size to less than 100
+- Cron Processing Time Limit should be set not more than 50 seconds to be safe with the 59 seconds platform PHP timeout.
+- Set the plugin to only process 1, this depends on how many post_meta's and custom functions are associated on every post imported, you can adjust it to a higher number that is optimal in your site if it will be a recurring import in order to speed it up but be reminded that it is a trial and error process which you need to experiment on works best from your end.
+- Instead of importing it as a one large file, it is best to set it up as reoccurring cron import as outlined [here]( https://www.wpallimport.com/documentation/recurring/cron/).
+- For busy sites while doing recurring cron, you can add cron sleep of at least 10 seconds to free up some php workers on recurring cron imports.
 
-- Splitting the import or export into smaller parts
-- Set the plugin to only process 1 or 2 records per iteration
-
-**Issue 2:** Uploading large import files hits the 59 second [timeout](/timeouts), or you're getting invalid file paths.
+**Issue 2:** Getting invalid file paths when importing / exporting on environments with multiple appservers like test and live.
 
 **Solution:** You can upload the import file directly to the plugin's designated writable path `wp-content/uploads/wpallimport/files/`. When creating a new import using `existing file`, the file uploaded should appear there as an option .
 
+Note: Invalid file paths are caused by the old infrastructure having different binding paths in every appservers. If you are in the old infrastructure, try to request in support to be migrated to the [Compute Optimized Environment](https://pantheon.io/docs/platform-considerations#compute-optimized-environments-coe). 
```

Comment:
We should look into this, as I think all/most users should be on COE at this point.

Reply:
Not really sure, it is just a suggestion on what possible things to look at if there issues in the import process